Source/WebCore/page/LocalFrame.cpp

```diff
@@ -147,12 +147,21 @@ static inline float parentTextZoomFactor(LocalFrame* frame)
     return parent->textZoomFactor();
 }
 
+static bool isRootFrame(const Frame& frame)
```

Comment:
Why do we need this function? It seems like we can just use `Frame::isRootFrame()`.

Reply:
Ah, right.  I need to move that logic to here instead of duplicating it.